### DISCUSSION TITLE
ARM64EC

### DISCUSSION THREAD

**Discussion by billywhizz** [jart/cosmopolitan#533] [General]
I am just reading about [ARM64EC](https://blogs.windows.com/windowsdeveloper/2021/06/28/announcing-arm64ec-building-native-and-interoperable-apps-for-windows-11-on-arm/). I am not expert in cosmopolitan-level OS level wizardry but keen to learn more and enjoying watching the progress. 

My question is - would ARM64EC mean a cosmopolitan binary would run on Windows 11 on ARM64 supported chips out of the box? Sorry if it's a dumb question.

**Comment by jart** [comment]
Correct. Cosmopolitan Libc binaries work great on Windows ARM computers. Here's proof: https://www.youtube.com/watch?v=CQ7Npip88Yw

**Reply by billywhizz** [reply]
:open_mouth:  with extra royalty and celine dion! the question soon is going to be where will this **not** run! :clap:
